electronics_lib/test_capacitor_generic.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+
+from electronics_abstract_parts import *
+import electronics_abstract_parts
+import electronics_model
+from . import *
+from . import Passives
+from .Passives import SmtCeramicCapacitorGeneric
+from .test_passive_common import *
+
+
+class CapacitorGenericTestTop(Block):
+  def __init__(self):
+    super().__init__()
+    self.dut = self.Block(SmtCeramicCapacitorGeneric(
+      capacitance=0.1 * uFarad(tol=0.2),
+      voltage=(0, 3.3) * Volt
+    ))
+    (self.dummya, ), _ = self.chain(self.dut.pos, self.Block(PassiveDummy()))
+    (self.dummyb, ), _ = self.chain(self.dut.neg, self.Block(PassiveDummy()))
+
+
+class BigCapacitorGenericTestTop(Block):
+  def __init__(self):
+    super().__init__()
+    self.dut = self.Block(SmtCeramicCapacitorGeneric(
+      capacitance=(50, 1000) * uFarad,
+      voltage=(0, 3.3) * Volt
+    ))
+    (self.dummya, ), _ = self.chain(self.dut.pos, self.Block(PassiveDummy()))
+    (self.dummyb, ), _ = self.chain(self.dut.neg, self.Block(PassiveDummy()))
+
+
+class HighVoltageCapacitorGenericTestTop(Block):
```

Comment:
I don't think you have voltage ratings as part of the cap selection process, right? If not, you can probably delete this.